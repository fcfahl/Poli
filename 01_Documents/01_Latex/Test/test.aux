\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\hbox to\@tempdima {\hfil }Summary}{i}{section*.1}}
\@writefile{toc}{\contentsline {section}{\hbox to\@tempdima {\hfil }Acknoledge}{ii}{section*.2}}
\@writefile{toc}{\contentsline {section}{\hbox to\@tempdima {\hfil }List of Figures}{iv}{chapter*.4}}
\@writefile{toc}{\contentsline {section}{\hbox to\@tempdima {\hfil }List of Tables}{v}{chapter*.5}}
\@writefile{toc}{\contentsline {section}{\numberline {0.1}Introduction}{1}{section.0.1}}
\newlabel{sec:intro}{{0.1}{1}{Introduction}{section.0.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {0.2}Methodology}{2}{section.0.2}}
\newlabel{sec:meth}{{0.2}{2}{Methodology}{section.0.2}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.2.1}materials}{2}{subsection.0.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{sources}{2}{section*.6}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Option caption}}{3}{figure.0.1}}
\newlabel{fig:gland30}{{1}{3}{Option caption}{figure.0.1}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces Option caption}}{4}{figure.0.2}}
\newlabel{fig:gland30_2}{{2}{4}{Option caption}{figure.0.2}{}}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces this is an optional caption, without reference}}{5}{table.0.1}}
\newlabel{tab:test table 1}{{1}{5}{this is an optional caption, without reference}{table.0.1}{}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces import table}}{5}{table.0.2}}
\newlabel{tab:xls table}{{2}{5}{import table}{table.0.2}{}}
