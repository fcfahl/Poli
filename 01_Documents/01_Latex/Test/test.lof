\contentsline {figure}{\numberline {1}{\ignorespaces Option caption}}{3}{figure.0.1}
\contentsline {figure}{\numberline {2}{\ignorespaces Option caption}}{4}{figure.0.2}
