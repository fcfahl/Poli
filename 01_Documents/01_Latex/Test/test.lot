\contentsline {table}{\numberline {1}{\ignorespaces this is an optional caption, without reference}}{5}{table.0.1}
\contentsline {table}{\numberline {2}{\ignorespaces import table}}{5}{table.0.2}
