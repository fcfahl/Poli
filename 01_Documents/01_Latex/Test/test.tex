\documentclass{report}

\usepackage{lipsum}
\usepackage[margin=1in,left=1.2in,includefoot]{geometry}

\usepackage{graphicx}
\usepackage{float}

\usepackage[hidelinks]{hyperref}

% header and foot stuff
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhead{}
\fancyfoot{}
\fancyfoot[R]{\thepage}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0.5pt}

\begin{document}

\begin{titlepage}

	\begin{center}
	
	\line(1,0){300}\\
	[0.25cm]
	\huge{\bfseries Title}\\
	[0.25cm]
	\line(1,0){300}\\
	[10cm]

	\end{center}

	\begin{flushright}
	
	\textsc{\large{addsdsd}}\\	
	
	\end{flushright}


\end{titlepage}

\pagenumbering{roman}

\section*{Summary}
\addcontentsline{toc}{section}{\numberline{}Summary}
this is summary
\cleardoublepage

\section*{Acknoledge}
\addcontentsline{toc}{section}{\numberline{}Acknoledge}
this is Acknoledge
\cleardoublepage

\tableofcontents
\thispagestyle{empty}
\cleardoublepage


\listoffigures
\addcontentsline{toc}{section}{\numberline{}List of Figures}
\cleardoublepage

\listoftables
\addcontentsline{toc}{section}{\numberline{}List of Tables}
\cleardoublepage


\setcounter{page}{1}

\pagenumbering{arabic}

\section{Introduction}\label{sec:intro}

\lipsum[1]\

\lipsum[2]\

\lipsum[3]\

\lipsum[4]\

\lipsum[5]\

\lipsum[6]\

\lipsum[7]\

\section{Methodology}\label{sec:meth}
\subsection{materials}

\subsubsection{sources}

\lipsum[8]\
\lipsum[9]\

Figure \ref{fig:gland30} shows land use\\

\lipsum[9]\

Figure \ref{fig:gland30_2} shows land use\\

\lipsum[10]\
\lipsum[11]\

the intro is found on page \pageref{sec:intro}


\begin{figure}
	\centering
	\includegraphics[scale=0.5]{/home/fcfahl/Documents/01_Latex/Test/GLand30.png}
	\caption[Option caption]{GLand30}
	\label{fig:gland30}
\end{figure}


\lipsum[10]\

\lipsum[11]\

\lipsum[10]\

\begin{figure}
	\centering
	\includegraphics[scale=0.5]{/home/fcfahl/Documents/01_Latex/Test/GLand30.png}
	\caption[Option caption]{GLand30}
	\label{fig:gland30_2}
\end{figure}

\lipsum[11]\

Table \ref{tab:test table 1} sjhows.....

\lipsum[10]\

\lipsum[11]\

\begin{table}[H]

	\centering
	\caption[this is an optional caption, without reference]{local caption, with reference}
	\label{tab:test table 1}
	\begin{tabular}{lcr}
		\bfseries{col1} & col2 & col3\\
		1 & a & q\\
		2 & b & a\\
		3 & c & s\\		
		4 & d & d\\
		5 & e & c\\		
		
	
	\end{tabular}

\end{table}

\lipsum[10]\

\lipsum[11]\

\begin{table}[H]

	\centering
	\caption{import table}
	\label{tab:xls table}
	\begin{tabular}{p{0.9in}p{0.9in}p{0.9in}}
		Theme & Topic1 & Topic2\\
		M1 & MODIS & LULC\\

		
	
	\end{tabular}

\end{table}


\end{document}