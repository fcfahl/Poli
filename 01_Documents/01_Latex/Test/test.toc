\contentsline {section}{\hbox to\@tempdima {\hfil }Summary}{i}{section*.1}
\contentsline {section}{\hbox to\@tempdima {\hfil }Acknoledge}{ii}{section*.2}
\contentsline {section}{\hbox to\@tempdima {\hfil }List of Figures}{iv}{chapter*.4}
\contentsline {section}{\hbox to\@tempdima {\hfil }List of Tables}{v}{chapter*.5}
\contentsline {section}{\numberline {0.1}Introduction}{1}{section.0.1}
\contentsline {section}{\numberline {0.2}Methodology}{2}{section.0.2}
\contentsline {subsection}{\numberline {0.2.1}materials}{2}{subsection.0.2.1}
\contentsline {subsubsection}{sources}{2}{section*.6}
