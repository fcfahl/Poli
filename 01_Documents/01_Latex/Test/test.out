\BOOKMARK [1][-]{section*.1}{Summary}{}% 1
\BOOKMARK [1][-]{section*.2}{Acknoledge}{}% 2
\BOOKMARK [1][-]{chapter*.4}{List of Figures}{}% 3
\BOOKMARK [1][-]{chapter*.5}{List of Tables}{}% 4
\BOOKMARK [1][-]{section.0.1}{Introduction}{}% 5
\BOOKMARK [1][-]{section.0.2}{Methodology}{}% 6
\BOOKMARK [2][-]{subsection.0.2.1}{materials}{section.0.2}% 7
